WebRoot/css/zm.css

```diff
@@ -4398,18 +4398,21 @@ HTML>BODY .appt-selected .appt_allday_body
 
 .ZmShortcutList {
 	padding: 1rem;
-	-moz-columns: 3;
-	-webkit-columns: 3;
-	columns: 3;
+	-moz-columns: 2;
```

Comment:
I am not exactly sure but we can use any of below macros, we need to check this

// operating systems
			define(macros, "LINUX", isLinux);
			define(macros, "MACINTOSH", isMac);
			define(macros, "WINDOWS", isWindows);

			// browser variants
			define(macros, "CHROME", isChrome);
			define(macros, "CHROME_4_OR_HIGHER", isChrome4up);
			define(macros, "FIREFOX", isFirefox);
			define(macros, "FIREFOX_1_OR_HIGHER", isFirefox1up);
			define(macros, "FIREFOX_1_5_OR_HIGHER", isFirefox1_5up);
			define(macros, "FIREFOX_4_OR_HIGHER", isFirefox4up);
			define(macros, "GECKO", isGeckoBased);
			define(macros, "GECKO_1_8_OR_HIGHER", isGecko1_8up);
			define(macros, "GECKO_2_OR_HIGHER", isGecko2up);
			define(macros, "HOTJAVA", isHotJava);
			define(macros, "IPHONE", isIPhone);
			define(macros, "MOZILLA", isMozilla);
			define(macros, "MOZILLA_1_4_OR_HIGHER", isMozilla1_4up);
			define(macros, "MSIE", isIE);
			define(macros, "MSIE_3", isIE3);
			define(macros, "MSIE_4", isIE4);
			define(macros, "MSIE_4_OR_HIGHER", isIE4up);
			define(macros, "MSIE_5", isIE5);
			define(macros, "MSIE_5_OR_HIGHER", isIE5up);
			define(macros, "MSIE_5_5", isIE5_5);
			define(macros, "MSIE_5_5_OR_HIGHER", isIE5_5up);
			define(macros, "MSIE_6", isIE6);
			define(macros, "MSIE_6_OR_HIGHER", isIE6up);
			define(macros, "MSIE_LOWER_THAN_7", isIE && !isIE7up);
			define(macros, "MSIE_7", isIE7);
			define(macros, "MSIE_7_OR_HIGHER", isIE7up);
			define(macros, "MSIE_8", isIE8);
			define(macros, "MSIE_8_OR_HIGHER", isIE8up);
			define(macros, "MSIE_LOWER_THAN_9", isIE && !isIE9up);
			define(macros, "MSIE_9", isIE9);
			define(macros, "MSIE_9_OR_HIGHER", isIE9up);
			define(macros, "MSIE_LOWER_THAN_10", isIE && !isIE10up);
			define(macros, "MSIE_10", isIE10);
			define(macros, "MSIE_10_OR_HIGHER", isIE10up);
			define(macros, "MODERN_IE", isModernIE);
			define(macros, "NAVIGATOR", isNav);
			define(macros, "NAVIGATOR_4", isNav4);
			define(macros, "NAVIGATOR_6", isNav6);
			define(macros, "NAVIGATOR_6_OR_HIGHER", isNav6up);
			define(macros, "NAVIGATOR_7", isNav7);
			define(macros, "NAVIGATOR_COMPATIBLE", isCompatible);
			define(macros, "OPERA", isOpera);
			define(macros, "SAFARI", isSafari);
			define(macros, "SAFARI_2", isSafari2);
			define(macros, "SAFARI_2_OR_HIGHER", isSafari2up);
			define(macros, "SAFARI_3", isSafari3);
			define(macros, "SAFARI_5_OR_HIGHER", isSafari5up);
			define(macros, "TRIDENT", isTrident);
			define(macros, "WEBKIT", isWebKitBased);
			define(macros, "WEBTV", isWebTv);